src/Backend/Modules/Tags/Engine/Model.php

```diff
@@ -289,79 +320,100 @@ public static function saveTags($otherId, $tags, $module, $language = null)
                 if ($tag == '') {
                     unset($tags[$key]);
                 } else {
-                    $tags[$key] = $tag;
+                    $tags[$key] = (string) $tag;
```

Comment:
Why do you cast it to string then? It should all be strings, even without casting it.